fastlane/metadata/default/release_notes.txt

```diff
@@ -0,0 +1,5 @@
+There’s a new block in town! With the Embed block, you can embed content from your favorite services like YouTube, TikTok, Instagram, and more. Stay tuned for additional functionality to come.
+
+Love our Blogging Reminders feature? Previously, only site admins could configure reminders, but we’ve now enabled this feature for people who are able to edit posts on your site (editors, authors, and contributors).
+
+We’ve made improvements to comment moderation, ensuring that the appropriate users can swipe left to approve/unapprove and trash comments. We’ve also provided support for Global Styles, enabling settings like color and background customization for Full Site Editing themes.
```

Comment:
Made me pause at first to see this diff for `default` appears here given that we merged the release notes on Friday (for them to make it to the Weekly Kit in time) so I expected them to already be in develop.

But in retrospect I think this is ok, because this `fastlane/metadata/default/` is what's downloaded/exported from GlotPress as the source (which actually confirms that the release notes are already in GP) – as you also mention [in your comment above](https://github.com/wordpress-mobile/WordPress-iOS/pull/16959#pullrequestreview-719772411) – and not what GP will import (which is the `.po` file we merged on Friday), so all is good here 👌 🙆‍♂️ :phew: